resources/icons.css

```diff
@@ -219,12 +219,14 @@
     height: 16px;
     background-image: url('icons.png') !important;
     background-repeat: no-repeat !important;
-    background-position: -554px 0px !important;
+    background-position: -552px 2px !important;
```

Comment:
The icons.css is a generated file (based on image-files in the resources/icons folder) so lets try finding another way to fix this instead.